engine/init.py

```diff
@@ -98,8 +99,10 @@ def init(
         logger.warning(warn_template.format("file_id", ""))
     if not reference:
         logger.warning(warn_template.format("reference", ""))
-    if not member_ids:
-        logger.warning(warn_template.format("member_ids", format_member_ids))
+    if not member_num:
+        logger.warning(warn_template.format("member_num", member_num))
+    if not member_type:
+        logger.warning(warn_template.format("member_type", member_type))
```

Comment:
We are using probtest in a way that we rarely require to set all of the variables. So probably all the warnings could be removed. It might be just useful in certain cases for developing, to have some safety checks. I have no strong opinion on this @faeglas

Reply:
I really think the warnings should stay. It's good to warn users about additional options that they have not set (maybe because they forgot or didn't know about).

Reply:
Thanks, so I leave it like it is!